src/Events/Integrations/Plugins/Event_Tickets/Emails/Email/RSVP.php

```diff
@@ -166,54 +169,4 @@ public function include_event_links( $parent_template ) {
 	public function include_event_link_styles( $parent_template ): void {
 		tribe( Template::class )->template( 'template-parts/header/head/tec-styles', $parent_template->get_local_values(), true );
 	}
-
-	/**
-	 * Determines whether RSVP should show calendar links.
-	 *
-	 * @since TBD
-	 *
-	 * @param array $args References template context arguments.
-	 *
-	 * @return bool
-	 */
-	public function should_show_links( $args ): bool {
-		// Double assigned due to needing to reference the original RSVP class later on.
-		$rsvp_class = $email_class = tribe( RSVP_Email::class );
-		if ( ! $email_class->is_enabled() ) {
-			return false;
-		}
-
-		$use_ticket_email = tribe_get_option( $email_class->get_option_key( 'use-ticket-email' ), false );
-		if ( ! empty( $use_ticket_email ) ) {
-			$email_class = tribe( Tickets_Email_Ticket::class );
-
-			if ( ! $email_class->is_enabled() ) {
-				return false;
-			}
-		}
-		if (
-			! empty( $args['email'] )
-			&& $args['email']->get_id() !== $rsvp_class->get_id()
-		) {
-			return false;
-		}
-
-		if ( ! empty( $args['preview'] ) && isset( $args['add_event_links'] ) ) {
```

Comment:
We must keep these checks for the preview to work as expected.

Reply:
I didn't find any issues with the previews, I will adding the screencast as artifact.